src/server.lisp

```diff
@@ -0,0 +1,184 @@
+;;;; server.lisp
+;;;;
+;;;; Author: Eric Peterson
+;;;;
+;;;; Lisp mimic of the python JSON RPC Server.
```

Comment:
Can we have a long-form comment on how all this stuff is supposed to work (what a dispatch table is, how it helps you solve your life problems, etc)?